pkg/agent/types/multicast.go

```diff
@@ -36,8 +36,7 @@ var (
 )
 
 type MulticastValidator interface {
```

Comment:
If it just for looking up the rule, name the interface McastNetworkPolicyController, or McastNetworkPolicyQuerier?

Reply:
done